### PR TITLE
Update README.md and Personal Portfolio 

### PR DESCRIPTION
**Pull request by evalee20000831**
I am unsure what information should I contain in README.md. Let me know if I should add more!

### CODE REVIEW

**Review by 0charleschen0 — APPROVED** [review]
LGTM! Approved!

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,55 @@
...
+        better person and motivated me to be a software engineer. It was 
+        because of this anime, I found what I wanted to do for future (at 
+        least for now)</p>
+        <p> If you want to know more about <i>Space Brothers</i>, please click 
+        <a href="https://en.wikipedia.org/wiki/Space_Brothers_(manga)"> 
+        here </a></p>
+        <p>Click here to get a random quote from my favorite anime
+            <i>Space Brothers</i>:</p>
+        <button onclick="addRandomQuote()">Quote</button>
+        <div id="quote-container"></div>
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/index.html]: Please keep +2 indentations
https://google.github.io/styleguide/htmlcssguide.html#General_Formatting_Rules

**Review by 0charleschen0 — COMMENTED** [review]
LGTM

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,24 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  // const variable cannot be reassigned. 
+  const quote1 = "An earnest failure has meaning.";
+  const quote2 = "Do what you can until you learn what you need. " + 
+                     "If you’re going to try something, you may as well" + 
+                     " aim for the top.";
+  const quote3 = "A leader must do more than lead.";
+  const quote4 = "When in doubt, don't worry about what's right. " + 
+                    "Your heard knows what's right, Choose the answer " +
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/script.js]: nit: When line-wrapping, put + at the beginning

File: portfolio/src/main/webapp/catImages.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/catImages.html:1]: LGTM

**Review by 0charleschen0 — COMMENTED** [review]
Create new folder for images and update 13 images for Alishan and cat html - LGTM

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -4,6 +4,134 @@
...
+    border: none;
+    color:rgb(255, 241, 40);
+    padding: 10px 25px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+}
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/style.css]: nit: +2 idents

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -4,6 +4,134 @@
...
+/* button used for github link */
+.button1:hover {
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/style.css]: Use self-explanatory naming, such as github-button. Same as below.

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -4,6 +4,134 @@
...
+  text-align: justify;
+  text-justify: inter-word;
+}
+
+body {
+  color: aliceblue; 
+  background-color:rgb(0, 0, 0);
+}
+
+body.two{
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/style.css]: a space

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -4,6 +4,134 @@
...
+    color:rgb(255, 241, 40);
+    padding: 10px 25px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+}
+/* button used for github link */
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/style.css]:
```suggestion
/* button used for github link */
```
Could be removed after renaming.

**Review by 0charleschen0 — COMMENTED** [review]
"Update index.html."

Add more contents about the change.
What are changes of index.html
ex:
- Add introduction image.
- Add my personal profile.
-

File: portfolio/src/main/webapp/education.html [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    <meta charset="UTF-8">
+    <title>Education</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body class="two">
+      <div id="content">
+          <div id="thirdBack">
+          <h1 style="color: yellow">Computer Science</h1>
+          
+
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/education.html]: nit: remove

File: portfolio/src/main/webapp/education.html [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+          <div id="thirdBack">
+          <h1 style="color: yellow">Computer Science</h1>
+          
+      
+          <h3>Bachelor of Computing Honour Computer Science Specialization 
+            - Artificial Intelligence Stream</h3>
+        <p class="a">sssss</p>
+        
+    </div>
+    </div>
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/education.html]: Please follow [coding style](https://google.github.io/styleguide/htmlcssguide.html#General_Formatting_Rules)

File: portfolio/src/main/webapp/education.html [hunk 10]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Education</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body class="two">
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/education.html]: Use more meaningful naming. This and below.

**Review by 0charleschen0 — COMMENTED** [review]
"Update pics."
Please explain what pics about in commit message.

**Review by 0charleschen0 — COMMENTED** [review]
"Update catImages.html."
- Please add more contents about the change ex: Add more content about catImages.html

File: portfolio/src/main/webapp/alishanImages.html [hunk 11]
```diff
@@ -2,14 +2,18 @@
...
     <div id="content">
-      <h1>Alishan Image Gallery</h1>
-      <div class="image-thumbnails">
+        <h1>Recent events - Alishan 🌿</h1> 
+        <h3>I just went to Alishan (阿里山) first time in life. Now I can 
+            finally say I am Taiwanese!</h3>
+
         <h3>Alishan Mianyue Line / 阿里山珉月線鐵路</h3> 
+        <a href="eva-images/IMAG6434.jpg"><img src="eva-images/IMAG6434.jpg" 
+        alt="Alishan-Landscape" width="302.4" height="403.2"/></a>
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/alishanImages.html]: Please follow the [coding style](https://google.github.io/styleguide/htmlcssguide.html#General_Formatting_Rules)

**Review by 0charleschen0 — COMMENTED** [review]
"Update alishanImages.html."
Please add more contents about this commits.

File: portfolio/src/main/webapp/education.html [hunk 12]
```diff
@@ -1,21 +1,100 @@
...
+                    <p class="e">Language</p>
+                    <hr>
+                    <ul class="a">
+                        <li>Fluent in English and Mandarin</li>
+  			            <li>JLPT N3~N2 level Japanese</li>
+  			            <li>Conversational level Taiwanese</li>
+		            </ul>
+                </div>
+            </div>
+        </div>
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/education.html]: Please follow [coding style](https://google.github.io/styleguide/htmlcssguide.html#General_Formatting_Rules)

**Review by 0charleschen0 — COMMENTED** [review]
"Finish up the portfolio."
Add more contents.

**Review by 0charleschen0 — COMMENTED** [review]
It should be "Fix grammar in Personal Portfolio".

File: portfolio/src/main/webapp/education.html [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+        <!-- LANGUAGE --> 
+        <div class="row">
+          <div class="column left">
+            <p class="section-title">Language</p>
+            <hr>
+            <ul class="course">
+              <li>Fluent in English and Mandarin</li>
+  			  <li>JLPT N3~N2 level Japanese</li>
+  			  <li>Conversational level Taiwanese</li>
+		    </ul>
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/education.html]: nit: idents

File: portfolio/src/main/webapp/script.js [hunk 14]
```diff
@@ -13,16 +13,24 @@
...
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  // const variable cannot be reassigned. 
+  const quote1 = "An earnest failure has meaning.";
+  const quote2 = "Do what you can until you learn what you need. "
+                     + "If you’re going to try something, you may as well" 
+                     + " aim for the top.";
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/script.js]:
```suggestion
  const quote2 = "Do what you can until you learn what you need. "
      + "If you’re going to try something, you may as well" 
      + " aim for the top.";
```

Comment (0charleschen0) [reply portfolio/src/main/webapp/script.js]: nit: +4 Idents

File: portfolio/src/main/webapp/script.js [hunk 15]
```diff
@@ -13,16 +13,24 @@
...
+  const quote3 = "A leader must do more than lead.";
+  const quote4 = "When in doubt, don't worry about what's right. "
+                    + "Your heard knows what's right, Choose the answer "
+                    + "that will be fun.";
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/script.js]: Same as above

File: portfolio/src/main/webapp/style.css [hunk 16]
```diff
@@ -1,9 +1,194 @@
...
+  left: 50%;
+  width: 80%; 
+  transform: translate(-50%, -50%);
+  padding: 10px;
+  color:rgb(255, 255, 255);
+  font-family: "Lucida Console", Courier, monospace;
+}
+
+/* middle part of index.html page */
+* {
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/style.css]: Could you explain why we have * here?

File: portfolio/src/main/webapp/style.css [hunk 17]
```diff
@@ -1,9 +1,194 @@
...
+  color:rgb(255, 241, 40);
+  padding: 10px 25px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+}
+.github-button:hover {
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/style.css:134]: nit: A empty line
```suggestion

.github-button:hover {
```

File: portfolio/src/main/webapp/style.css [hunk 18]
```diff
@@ -1,9 +1,194 @@
...
+  background-color:rgb(255, 241, 40);
+  color:rgb(24, 21, 21);
+}
+
+.fb-button {
+  background-color: rgb(43, 90, 219); 
+  color: rgb(255, 255, 255); 
+}
+.fb-button:hover {
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/style.css:144]: Same here.

File: portfolio/src/main/webapp/style.css [hunk 19]
```diff
@@ -1,9 +1,194 @@
...
+}
+
+.link-button {
+  background-color: rgb(0, 0, 0); 
+  color: rgb(255, 255, 255); 
+  padding: 15px 25px;
+  border: 2px solid rgb(255, 255, 255);
+  transition-duration: 0.4s;
+}
+.link-button:hover {
```
Comment (0charleschen0) [inline portfolio/src/main/webapp/style.css:157]: And here.

**Review by joeythegod — APPROVED** [review]
LGTM. For html part I think next time you can use template extending, you make a base “skeleton” template and it can reduce your code significantly :)

**Review by teefang — APPROVED** [review]

**Pull request merged**